docs/getting-started/literature.md

```diff
@@ -9,111 +9,152 @@ image: https://user-images.githubusercontent.com/78981416/215575718-9d206b3c-437
-
-- Offers an ["Auto Colors"](https://user-images.githubusercontent.com/78981416/233718417-d994059b-b18d-4fa1-92cf-e2f7a51bd072.png) option which allows you to fix black levels, however requires manual tweaking depending on the source. Should only be enabled when [obvious blacks are appearing as grey](https://user-images.githubusercontent.com/78981416/233718453-c358222b-384e-45f5-9ff5-151aad32c94f.png), at which point you should increase the white level sensitivity slightly [until they become black](https://user-images.githubusercontent.com/78981416/233718539-a670d966-9ab7-4f23-8abd-a1a8f2cb93f4.png).
-
-[Tachidesk](https://github.com/Suwayomi/Tachidesk-Server) (Most Convenient)
-
-- Tachiyomi fork for PC
-
-#### Android
-
-[Perfect Viewer](https://play.google.com/store/apps/details?id=com.rookiestudio.perfectviewer) (Highest Quality)
-
-- Standalone reader that requires you to bring your own manga, provides the [highest quality](https://slow.pics/c/y737QBlP) scaling available on Android with Image smooth filter set to [Lanczos 3](https://user-images.githubusercontent.com/78981416/233718601-dbdd3303-d96a-474e-aed5-7d4c22a8e8da.png)
-
-[Tachiyomi](https://tachiyomi.org/) (Most Convenient)
-
-- All in one package with sourcing, tracking and a reader.
-  In settings -> reader enable 32 bit color to [avoid banding](https://slow.pics/c/eOC7j5nI)
-  Under Browse, add Mangasee and Mangadex. The former will provide official rips whilst the latter will provide scanlations.
-
-[TachiyomiSY](https://tachiyomi.org/forks/TachiyomiSY/)/[TachiyomiJ2K](https://tachiyomi.org/forks/TachiyomiJ2K/)/[TachiyomiAZ](https://tachiyomi.org/forks/TachiyomiAZ/)/[Neko](https://tachiyomi.org/forks/Neko/)
-
-- Tachiyomi forks with added features.
-
-#### iOS
-
-[Sidebooks](https://apps.apple.com/us/app/sidebooks/id409777225) (Highest Quality)
-
-- Standalone reader that requires you to bring your own manga, provides the [highest quality](https://slow.pics/c/gUsyOomL) scaling available on iOS.
-
-[Paperback](https://paperback.moe/) (Most Convenient)
-
-- Tachiyomi alternative that supports adding sources.
-
-#### Browser (Any platform)
-
-[MangaDex](https://mangadex.org/) (Most Convenient Scanlation)
-
-- Most scanlators upload here, which are then taken by aggregator sites. However, not everything is uploaded and official translations are never uploaded here.
-
-[Mangasee](https://mangasee123.com/) or [Manga4life](https://manga4life.com/) (Most Convenient Official)
+### Sourcing
 
-- These sites upload downscaled official rips with the highest quality among online sites. The original rips can be found on nyaa.
++++ Downloads
 
-### Organization and Manga Servers
+[Nyaa](https://nyaa.si) [!badge icon=":heart:" variant="primary" text="Recommended"]
+:   Public torrent tracker with the most scanlations and official digital rips
```

Comment:
should mention quality difference, also nyaa may have less scanlations than typical manga aggregator sites